internal/graphql/model/operator_helper.go

```diff
@@ -106,7 +106,7 @@ func GetBrokerInfo(mesheryclient operatorClient.Interface, mesheryKubeClient *me
 	}
 	if err == nil {
 		status := fmt.Sprintf("%s %s", StatusConnected, broker.Status.Endpoint.External)
-		if brokerConn.Info() == brokerpkg.NotConnected {
```

Comment:
`brokerConn.Info()` returns `meshery` when connected otherwise `not-connected`.

Reply:
Thank you, @MUzairS15